packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -390,13 +461,7 @@ export default class Meeting extends StatelessSparkPlugin {
    * @memberof Meeting
    */
   leave() {
-    return this.meetingRequest.leaveMeeting({
-      locusUrl: this.locusUrl,
-      selfId: this.selfId,
-      correlationId: this.correlationId,
-      resourceId: null,
-      deviceUrl: this.deviceUrl
-    });
+    return MeetingUtil.leaveMeeting(this);
```

Comment:
I don't really love passing `this` as an entirety to another util file. Maybe personal preference

Reply:
ya, agreed, it looks off, but it's actually just the meeting object, and it should have access to it's properties to leave the meeting. if we want to specify exactly the params to pass we can do that as well, but I wanted to get used to passing the whole meeting object around rather than it's random values in a variety of places. everything will just deal with a meeting object